### ISSUE TITLE
Bug: Imposter transforms variables with `namespace` keyword

### ISSUE THREAD

**Issue by dmytro-kovalov** [TypistTech/imposter#79]
Hi,

While working with [sabre/xml](https://packagist.org/packages/sabre/xml) package I found that Imposter transforms variables with `namespace` keyword, used as a function argument or just in code.

For example,
```
function dummyFunction(string $namespace = null) : array {
  // code
}
```

becomes:
```
function dummyFunction(string $namespace MyPlugin\Vendor\= null) : array {
  // code
}
```

which caused PHP Fatal error (syntax error).

![screenshot](https://i.imgur.com/GXmLy2U.png)

**Comment by mundschenk-at** [comment]
Not only variables, also comments that contain the word "namespace".

**Comment by EvanShaw** [comment]
Can confirm, same thing happened to me. It wasn't immediately obvious either what the problem was since autoloading fails halfway through and PHP simply reports 'Class not found' during runtime.

**Comment by Kubitomakita** [comment]
Same here.

```php
$_namespace = var_export($ns, true);
```

Gets replaced with

```php
$_namespace Prefixed\Namespace\Vendor\= var_export($ns, true);
```

**Comment by TangRufus** [comment]
Sorry for the delay.

It should be fixed by #82 
Can you guys test it (`dev-master`)?

Re-open this issue if the problem persists

**Comment by Kubitomakita** [comment]
Seem to work well in my case now!
